api/src/services/listing-csv-export.service.ts

```diff
@@ -348,6 +348,16 @@ export class ListingCsvExporterService implements CsvExporterServiceInterface {
         format: (val: string): string =>
           formatLocalDate(val, this.dateFormat, this.timeZone),
       },
+      {
+        path: 'copyOfId',
+        label: 'Copy or Original',
+        format: (val: string): string => (val ? 'Copy' : 'Original'),
+      },
+      {
```

Comment:
I think since both this and the below have the same `path` key->value pair it will likely cause problems with the way the spreadsheet export wants to work (for when we get there)